packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.tsx

```diff
@@ -34,8 +42,26 @@ const dataTipText2 = `The repeatable zone is for a group<br/>
 
 const FieldZones: FC = () => {
   const { slice, setSlice, variation } = useSliceState();
+  const [
+    isDeleteRepeatableZoneDialogOpen,
+    setIsDeleteRepeatableZoneDialogOpen,
+  ] = useState(false);
+
+  // We won't show the Repeatable Zone if no items are configured.
+  const hasItems = Boolean(
+    variation.items && Object.keys(variation.items).length > 0,
+  );
 
   const _onDeleteItem = (widgetArea: WidgetsArea) => (key: string) => {
+    if (
+      widgetArea === WidgetsArea.Items &&
+      variation.items &&
+      Object.keys(variation.items).length <= 1
+    ) {
+      setIsDeleteRepeatableZoneDialogOpen(true);
```

Comment:
✨ _love it_

That's super smart to open the modal and explain what's happening!
I guess it's something you can share with Côme and Benjamin, It's a very good idea.

Reply:
Good idea! I asked them here: https://prismic-team.slack.com/archives/C02G5CFFLR2/p1712869312778609